db/index.js

```diff
@@ -0,0 +1,14 @@
+const { Pool } = require("pg"); 
```

Comment:
I'm not sure why you have created this file when there is already a connection.js file with this information

Reply:
oh yes thank you i didnt realise we were provided one already :) i have now deleted the duplicate file as you suggested